pandas/tests/io/test_pickle.py

```diff
@@ -283,7 +283,7 @@ def compress_file(self, src_path, dest_path, compression):
             raise ValueError(msg)
 
         if compression not in ["zip", "tar"]:
-            with open(src_path, "rb") as fh, f:
+            with open(src_path, "rb") as fh:
                 f.write(fh.read())
```

Comment:
does it work to nest the with statements? like
```python
with open(src_path, 'rb') as fh:
    with f:
        f.write(fh.read())
```
?

Reply:
Thanks, @MarcoGorelli, it works like a charm. I updated the commit.